scripts/prepare_sector_network.py

```diff
@@ -3732,6 +3751,8 @@ def lossy_bidirectional_links(n, carrier, efficiencies={}):
     )
     pop_weighted_energy_totals.update(pop_weighted_heat_totals)
 
+    capacities_OCGT, efficiencies_OCGT = get_capacities(n, carrier="OCGT", component="generators")
```

Comment:
maybe we shouldn't make it a default, but make it configurable? i.e. in config.default define a new entry in `sector`: `keep_OCGT: False` which is False by default and then something like
```suggestion
    if options["keep_OCGT"]:
        capacities_OCGT, efficiencies_OCGT = get_capacities(n, carrier="OCGT", component="generators")
    else:
        capacities_OCGT, efficiencies_OCGT = 0, None
```

(please test before comitting, I just made it up). This would ensure that the current workflow isn't changed